nomos-cli/src/cmds/disseminate/mod.rs

```diff
@@ -19,8 +19,8 @@ pub struct Disseminate {
     #[clap(short, long)]
     pub network_config: PathBuf,
     /// The data availability protocol to use. Defaults to full replication.
-    #[clap(flatten)]
-    pub da_protocol: DaProtocolChoice,
+    // #[clap(flatten)]
+    // pub da_protocol: DaProtocolChoice,
```

Comment:
Leftover? Probably we just want the real stuff. Full-replication should be used for mocking testing purposes only (?).